src/App.jsx

```diff
@@ -9,7 +11,10 @@ const article = {
 };
 
 const App = () => (
-  <h1>React Article</h1>
+  <>
+    <h1>React Article</h1>
+    <Article title={article.title} text={article.text} date={article.date} />
```

Comment:
What do we do when we have 3 more than 3 attributes? We put them eac on a new line
```suggestion
    <Article 
      title={article.title} 
      text={article.text} 
      date={article.date} 
    />
```